### ISSUE TITLE
Change "Server" tab channel list into a popup when joining channels

### ISSUE THREAD

**Issue by gajop** [Spring-Chobby/Chobby#87]
https://github.com/Spring-Chobby/Chobby/blob/master/LuaUI/widgets/chobby/components/chat_windows.lua#L11-L47

& other related code should be removed, along with the "Server" tab.

Querying for a Server list is unnecessary and resource intensive for the server. Having one less Tab also saves room, and channels can easily be joined by typing their name

**Comment by GoogleFrog** [comment]
Just make it a debug option. Also, we need tab panel to support having 0 tabs

**Comment by gajop** [comment]
Actually I've changed my mind on this. It could be made into a popup, showing the user a list of suggested channels. The tab should still be removed.
PS: This is unrelated to the Debug tab, which should just be a debug option.

**Comment by GoogleFrog** [comment]
I don't think we need server tab. Perhaps we could expand the "join channel" button to contain suggested channels. However, it we were to do that why not just autojoin those channels?

**Comment by gajop** [comment]
I don't think "server" should be a tab either.

Yes, I think we should extend the button, and it's something I've seen a few games do. Essentially have the join channel button also provide a list of all public server channels sorted by number of users. This might expose the user to bits and pieces of the community they didn't know about.
I've seen some servers also curate this list, but I don't think we have the time to do that.

Outside of server specific main channels, e.g. "#zk", and "#main", and perhaps "#newbies" for new players, I strongly dislike autojoining everything potentially useful.

Generally I think most users won't be joining channels. Usually people just chat with friends, team members and the default channel. So having a list of popular channels may give them a hint what they could join and perhaps increase their interest.
